packages/odyssey-storybook/src/components/odyssey-mui/Autocomplete/Autocomplete.stories.tsx

```diff
@@ -278,6 +307,18 @@ export const Loading: StoryObj<AutocompleteType> = {
     isLoading: true,
     options: [],
   },
+  play: async ({ canvasElement, step }) => {
+    const canvas = within(canvasElement);
+    const comboBoxElement = canvas.getByRole("combobox") as HTMLInputElement;
+    await step("Click for loading to be visible", async () => {
+      await userEvent.click(comboBoxElement);
+      const presentationElement = screen.getByRole("presentation");
+      await expect(presentationElement).toBeVisible();
+      await expect(presentationElement.firstChild?.textContent).toBe(
+        "Loading…",
+      );
+    });
```

Comment:
Is there a reason `textContent` wouldn't exist on `firstChild`? If it doesn't exist, I think something's wrong with the browser unless `.firstChild` is `null`, in which case, we do want to error here.

```suggestion
    await step("Click for loading to be visible", async () => {
      await userEvent.click(comboBoxElement);
      const presentationElement = screen.getByRole("presentation");
      await expect(presentationElement).toBeVisible();
      await expect(presentationElement.firstChild.textContent).toBe(
        "Loading…",
      );
    });
```

Is there a reason the loading indicator would disappear?

---

I don't think we need to select `firstChild.textContent` either. That bypasses Testing Library. Instead, just do this:
```suggestion
    await step("Click for loading to be visible", async () => {
      await userEvent.click(comboBoxElement);
      const presentationElement = screen.getByRole("presentation", { name: "Loading..." });
      await expect(presentationElement).toBeVisible();
    });
```

If the text's not there, our element selection will be `null`. This way, we let Testing Library figure out the text in there rather than the computer. Using `.firstChild` says "this is in the DOM", not "accessibility controls can see it".

---

But also `role="presentation"` means its not visible to screen readers. That's the point if the role. Do we want to have that "Loading..." text hidden? I would think it's important for someone to know they're waiting on info.

[`aria-busy`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-busy) exists. I think we're supposed to use that to show something's in a loading state. In that case, we wouldn't even be checking for a presentational element as none would exist. What we'd wanna know is if it's in a "busy" state with the "Loading..." text.

Reply:
```
screen.getByRole("presentation", { name: "Loading..." });
```
Does not work because there is another div in between `role="presentation"` and `Loading...` text.
I replaced it to `screen.getByText("Loading…");`

Reply:
I checked into this story, the "Loading..." text seems to be an English leak from MUI. That's unrelated to your changes, but also, it's the text shouldn't go away either. The PR has `isLoading` set to `true`. There's no logic to unset it for this story.

Reply:
Good found for `Loading...` is not localized. That also lead to `no options` is also missing localization.
The `Loading...` text handled by autocomplete and it should be the default behavior. 
There is a `isLoading` controls so it can be controlled. If set from false to true, it needs to click on the autocomplete to show the text again.